internal/pkg/service/buffer/cleanup/node_test.go

```diff
@@ -110,7 +110,7 @@ func TestCleanup(t *testing.T) {
 	assert.NoError(t, schema.Records().ByKey(recordKey1).Put("rec").Do(ctx, client))
 
 	// Add received stats for the cleaned-up slice
-	assert.NoError(t, statsRepo.Insert(ctx, "node-123", []statistics.PerAPINode{
+	assert.NoError(t, statsRepo.Put(ctx, []statistics.PerSlice{
 		{
```

Comment:
Predtym bola `slice` spolocna pre vsetky API nodes (kedze bola v `etcd`) a iba statistiky sa viedli samostatne `PerAPINode`. 

Teraz ma kazdy node/volume svoju `slice`. Takze aj statistiky zbierame `PerSlice`.
Je to vlastne o dost jednoduchsie.